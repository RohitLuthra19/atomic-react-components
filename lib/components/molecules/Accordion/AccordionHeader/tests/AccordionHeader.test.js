import React from 'react';
import { shallow, render } from 'enzyme';

import { AccordionHeaderVanilla } from '../index';

describe('<AccordionHeader />', () => {
  let AccordionHeaderComponent = '';
  beforeEach(() => {
    AccordionHeaderComponent = shallow(<AccordionHeaderVanilla>Test</AccordionHeaderVanilla>);
  });

  test('should render', () => {
  	expect(AccordionHeaderComponent.exists()).toBe(true);
  });

  test('should render correctly', () => {
    expect(AccordionHeaderComponent).toMatchSnapshot();
  });
});
