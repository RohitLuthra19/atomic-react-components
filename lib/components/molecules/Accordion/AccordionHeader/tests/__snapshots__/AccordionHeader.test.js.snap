// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccordionHeader /> should render correctly 1`] = `
<summary
  className=""
>
  Test
</summary>
`;
